laplace/baselaplace.py

```diff
@@ -1568,8 +1567,1025 @@ def sample(self, n_samples=100, generator=None):
 
 
 class FunctionalLaplace(BaseLaplace):
-    pass
+    """
```

Comment:
There shouldn't be `\n` after `"""`: https://peps.python.org/pep-0257/#multi-line-docstrings